unst_msh_gen/README.md

```diff
@@ -0,0 +1,42 @@
+# Project Name
+Unstructured mesh generation for WW3 using JIGSSAW.
+## Description
+Mesh generation script capable of creating unstructured meshes for WW3 global modeling. The tool leverages JIGSAWPY (https://github.com/dengwirda/jigsaw-python) for efficient triangulation.
+Main changes include:
+Implementation of the ocn_ww3.py to create uniform unstructured mesh for global WW3 model.
+This tool is under active development, with future work focused on variable unstructured mesh generation.
+
+## Installation
+
+1- Install jigsawpy (https://github.com/dengwirda/jigsaw-python)
+2- you need following packages:
+	numpy
+        scipy
+	packaging
+	netcdf4
+3- get the DEM:
+	$wget https://github.com/dengwirda/dem/releases/download/v0.1.1/RTopo_2_0_4_GEBCO_v2023_60sec_pixel.zip
+4- unzip the DEM 
+	unzip *.zip
+ 
+
+
+## Usage
+1- clone the repo
+	$mkdir meshgen
+	$cd meshgen
+	$git clone https://github.com/AliS-Noaa/WW3-tools/tree/add-unst-mesh-gen
+2- get the DEM:
+        $wget https://github.com/dengwirda/dem/releases/download/v0.1.1/RTopo_2_0_4_GEBCO_v2023_60sec_pixel.zip
```

Comment:
Can you make a note about where this file is expected to be in relation to the code?   Unless things have changed, the *.nc file that comes from this needs to be in a specific place which is not included in the instructions as of now.